Gothic-Unity-Project/Gothic-Unity
# Added dependency for HVR-DeviceSimulator

I added a package which offers a Simulator for HVR. It comes auto-loaded via Package manager. We only need to briefly check if both options of Simulator work from FeatureFlags.

**To Test**:
1. Start the game with FeatureFlags like this and check if you can WASD around (XRIT Simulator)

![image](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/92402588-99dc-4e3b-be9a-6df3f325eb53)

![XRIT-simulator](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/7475d149-0911-4567-8bf0-3bea07880694)

---

2. Start the game with HVR FeatureFlags and do the same WASD thing (Hint: You spawn inside the cellar, that's a known issue and unrelated to the simulator)
![image](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/69fb3893-2a15-4eae-ade6-308e86e12c12)

![HVR-simulator](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/489c2a9f-f15e-4ad8-83ba-fffd87305fea)


Hints:
* If you like, you can check this video for further usage details: https://www.youtube.com/watch?v=-LRJt0cD4xk
* Implementation detail: I created another GitHub repository which we include via Package Manager. Therefore XRIT users or HVR users never need to download anything from scratch. Gothic-UnZENity is taking care already. More details can be found here: https://github.com/Gothic-UnZENity-Project/hvr-simulator
